### PR TITLE
Add PHPStan

### LINKED ISSUE #4016: Add PHPstan in project and CI
PHPstan is a very powerful tool that does static code analysis. It is highly recommended for this project. 

Attention, it is compatible only from php7. It is therefore an addition that must be made in the roadmap.

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/4016
https://phpstan.org/

```sh
composer run-script phpstan
```

### CODE REVIEW

**Comment by Alkarex** [comment]
There are already a number of fixes to do to pass level 1, and then we can increase the rule-level for more https://phpstan.org/user-guide/rule-levels

Help welcome in this PR already to pass level 1.

I will then add it to our CI

File: p/api/greader.php [hunk 1]
```diff
@@ -1116,8 +1116,7 @@ function markAllAsRead($streamId, $olderThanId) {
 		case 'unread-count':
 			$output = isset($_GET['output']) ? $_GET['output'] : '';
 			if ($output !== 'json') notImplemented();
-			$all = isset($_GET['all']) ? $_GET['all'] : '';
-			unreadCount($all);
+			unreadCount();
```
Comment (Frenzie) [inline p/api/greader.php:1119]: unreadCount didn't do anything with the parameter?

**Review by Frenzie — APPROVED** [review]
Looks like it should be fine, but didn't check in the closest detail. Definitely needs full testing.

File: p/api/greader.php (hunk 1 above)
Comment (math-GH) [reply p/api/greader.php:1119]: it has no parameter

https://github.com/FreshRSS/FreshRSS/blob/ba48733f17006b5e07625a22811594e62033fcc8/p/api/greader.php#L443-L446

**Comment by Alkarex** [comment]
(off-topic) @math-GH You should have received an email from me 📬

File: app/Models/UserDAO.php [hunk 2]
```diff
@@ -5,9 +5,10 @@ public function createUser() {
 		require(APP_PATH . '/SQL/install.sql.' . $this->pdo->dbType() . '.php');
 
 		try {
-			$sql = $SQL_CREATE_TABLES . $SQL_CREATE_TABLE_ENTRYTMP . $SQL_CREATE_TABLE_TAGS;
+			$sql = $GLOBALS['$SQL_CREATE_TABLES'] . $GLOBALS['SQL_CREATE_TABLE_ENTRYTMP'] . $GLOBALS['SQL_CREATE_TABLE_TAGS'];
```
Comment (Alkarex) [inline app/Models/UserDAO.php:8]: Typo fixed in https://github.com/FreshRSS/FreshRSS/pull/4116

**Pull request merged**